### PR TITLE
Add Code of Conduct to our docs

### PR DESCRIPTION
**Pull request by jvanbaarsen**
This adds the CoC in a central place that will allow us to link to it
from all our public repo's.

### CODE REVIEW

File: source/appsignal/contributing.html.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ Our main projects include:
 Other projects we have open sourced, and are used internally by other projects,
 include:
 
-- [AppSignal Rust agent][appsignal-rust]  
+- [AppSignal Rust agent][appsignal-rust]
   Early stage proof of concept of AppSignal Rust integration.
-- [sql_lexer][sql_lexer]  
+- [sql_lexer][sql_lexer]
   Rust library to lex and sanitize SQL queries.
-- [probes.rs][probes-rs]
```
Comment (tombruijn) [inline source/appsignal/contributing.html.md]: ah the good old trim whitespace at end of line editor default.

Great! But not for markdown!

File: source/appsignal/code-of-conduct.html.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: Contributor Covenant Code of Conduct
```
Comment (tombruijn) [inline source/appsignal/code-of-conduct.html.md]: can we not simply say "Code of Conduct"?
```suggestion
title: Code of Conduct
```

File: source/appsignal/contributing.html.md [hunk 3]
```diff
@@ -105,3 +111,5 @@ network or integrations, please let us know immediately by sending an email to
 [github-appsignal]: https://github.com/appsignal
 [travis-appsignal]: https://travis-ci.org/appsignal
 [semver]: http://semver.org/
+[coc-contact]: mailto:contact@appsignal.com
+[coc]:/appsignal/code-of-conduct.html
```
Comment (tombruijn) [inline source/appsignal/contributing.html.md]:
```suggestion
[coc]: /appsignal/code-of-conduct.html
```

File: source/appsignal/contributing.html.md (hunk 1 above)
Comment (jvanbaarsen) [reply source/appsignal/contributing.html.md]: Why would that be a problem? They still show up as spaces right?

Comment (jvanbaarsen) [reply source/appsignal/contributing.html.md]: So you actually want to have them on a new line? Let me try and figure out how to re-add them, and disable this behaviour in vim for markdown.

Comment (tombruijn) [reply source/appsignal/contributing.html.md]: You probably need to configure the Ale fixer to not become active for certain file types. Not sure if that's built in or if you need to configure an override.

I'll fix it for now and merge the PR.

If you're interested, I use this to trim the whitespace:

https://github.com/tombruijn/dotfiles/blob/3fbd1645b4e0f32ecac43a54bb4392b84e22a9bd/vim/.vim/config/trailing_whitespace.vim#L15-L40

With the exception for markdown files at line 18:

https://github.com/tombruijn/dotfiles/blob/3fbd1645b4e0f32ecac43a54bb4392b84e22a9bd/vim/.vim/config/trailing_whitespace.vim#L18

**Pull request merged**